astroquery/jplhorizons/core.py

```diff
@@ -1181,14 +1224,23 @@ def _parse_result(self, response, verbose=None):
         H, G = nan, nan
         M1, M2, k1, k2, phcof = nan, nan, nan, nan, nan
         headerline = []
+        centername = ''
         for idx, line in enumerate(src):
             # read in ephemerides header line; replace some field names
             if (self.query_type == 'ephemerides' and
                     "Date__(UT)__HR:MN" in line):
                 headerline = str(line).split(',')
                 headerline[2] = 'solar_presence'
-                headerline[3] = 'flags'
+                if 'Earth' in centername:
+                    headerline[3] = 'lunar_presence'
+                else:
+                    headerline[3] = 'interfering_body'
                 headerline[-1] = '_dump'
+                if (
+                    isinstance(self.id, dict) or str(self.id).startswith('g:')
+                ):
```

Comment:
We have longer than 80 char lines, do keep this in one line instead of 1 with information and 2 of fillers. This is for here and a few other places, too.

Reply:
copy that.